### ISSUE TITLE
Transfer all data to OSF

### ISSUE THREAD

**Issue by rempsyc** [rempsyc/busara_dashboard#53]
GitHub has a usual repo size limit of 5GB and we're already at +3GB even before adding all the other journals we would like in the future. So I think a more scalable solution (and better also for open science) would be to manually store all the data on an OSF project, and then find a way to read the data directly from OSF in the automatic GitHub workflow, download the new data locally, and then merge the data. This way, (1) the repo size remains small and (2) the data is available in a conventional way (OSF) for others who wish to explore or investigate the data.

**Comment by rempsyc** [comment]
@psforscher I recommend that you [subscribe](https://github.com/rempsyc/busara_dashboard) to all new activity on this repo so you can see when I open new issues without me having to tag you :P (if you want)

<img src="https://github.com/user-attachments/assets/3f465749-a3ae-414b-95c7-238f2b7afd73" width="35%">

**Comment by psforscher** [comment]
haha makes sense, done!

**Comment by rempsyc** [comment]
The maximum GitHub file size is 100 MB, but our data alone (when joined in a single file) is +300 MB, so I am not able to push it to GitHub. That's a problem because the file is also too big to be included on shinyapps.io. So I created an OSF project where I manually put the current data for testing purposes. This is our repo: https://osf.io/unxrd/

**Comment by psforscher** [comment]
Makes sense! I think this is also what a colleague of mine (Erin Buchanan) did for a project we're working on related to stereotype threat.
